src/components/visualization/mermaidViz/mermaidViz.css

```diff
@@ -0,0 +1,16 @@
+.rustic-mermaid-container {
+  height: 100%;
+  width: 100%;
+}
+
+.rustic-mermaid-container div:last-of-type {
+  flex: 1;
+  min-height: 0;
+  display: flex;
+  justify-content: center;
+}
+
+.rustic-mermaid-container div:last-of-type svg {
+  height: 100%;
+  width: 100%;
+}
```

Comment:
we can use a custom class name. Also, its not clear why we are setting min-height to 0

Reply:
I will use a custom className. For `min-height: 0`, I can't make the diagram's height responsive without it.